code/model/UserDefinedForm.php

```diff
@@ -879,4 +728,44 @@ public function finished()
             'Form' => '',
         ));
     }
+
+	/**
+	 * Writes
+	 * @param $watch
+	 *
+	 * @return string
+	 */
+	private function BuildWatchJS($watch)
```

Comment:
Generally we try and follow a camel casing (PSR-2 compatible) method naming structure. There can be exceptions when public methods are required to be named with a capital letter to start for template reasons, but since this is a private method it should probably be camel cased.